pkg/geo/geomfn/distance.go

```diff
@@ -333,6 +380,9 @@ func (u *geomMinDistanceUpdater) IsMaxDistance() bool {
 	return false
 }
 
+// TwistGeometries implements the geodist.DistanceUpdater interface.
+func (u *geomMinDistanceUpdater) TwistGeometries() {}
```

Comment:
panic here with unimplemented.

Reply:
Sure,
If I understand it right we have to write a panic message like,
```
panic("Unimplemented ...")
```
wouldn't it make `ST_distance`, `MinDistance` inexecutable for a while?

Reply:
oh, you use this for all cases, my bad.
can you add a comment that this is unimplemented pending `ST_ShortestLine` / #49035 for geomfn, and for geogfn adding a  comment that this is unimplemented as we do not care about ordering.